<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="WKWZTywrXeq4hFI" failures="1" tests="4" name="testing.SmokeTest" time="13.406" errors="0" timestamp="2022-11-23T15:53:13 IST" skipped="0">
  <testcase classname="testing.SmokeTest" name="verifySocialMediaCount" time="3.489">
    <failure message="expected [5] but found [4]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [5] but found [4]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:907)
at org.testng.Assert.assertEquals(Assert.java:917)
at testing.SmokeTest.verifySocialMediaCount(SmokeTest.java:37)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- verifySocialMediaCount -->
  <system-out/>
  <testcase classname="testing.SmokeTest" name="verifyTitle" time="3.317"/>
  <system-out/>
  <testcase classname="testing.SmokeTest" name="verifyForgotPasswordLink" time="3.307"/>
  <system-out/>
  <testcase classname="testing.SmokeTest" name="verifyURL" time="3.293"/>
  <system-out/>
</testsuite> <!-- testing.SmokeTest -->
